### PR TITLE
Change scope localstorage

### PR DESCRIPTION
**Pull request by ryanmelt**
Locks a scope to a tab, but allows other tabs to be on a different scope but sampling scope, instead of always pulling from localStorage.

### CODE REVIEW

**Comment by ryanmelt** [comment]
closes #110

File: openc3-init/plugins/packages/openc3-tool-common/src/tools/base/AppNav.vue [hunk 1]
```diff
@@ -177,6 +177,18 @@ export default {
       }
     )
   },
+  methods: {
+    newTabUrl(tool) {
+      let url = null
+      if (tool.url[0] == '/' && tool.url[1] != '/') {
```
Comment (jmthomas) [inline openc3-init/plugins/packages/openc3-tool-common/src/tools/base/AppNav.vue:183]: How would the url[1] character ever be '/'?

File: openc3-init/plugins/packages/openc3-tool-common/src/tools/base/AppNav.vue [hunk 2]
```diff
@@ -177,6 +177,18 @@ export default {
...
+        url = new URL(tool.url, window.location.origin)
+      } else {
+        url = new URL(tool.url)
+      }
+      url.searchParams.set('scope', window.openc3Scope)
```
Comment (jmthomas) [inline openc3-init/plugins/packages/openc3-tool-common/src/tools/base/AppNav.vue:188]: searchParams?

**Review by jmthomas — COMMENTED** [review]
This all looks good. I assume there will be changes to the Enterprise as well?

File: openc3-init/plugins/packages/openc3-tool-common/src/tools/base/AppNav.vue (hunk 1 above)
Comment (ryanmelt) [reply openc3-init/plugins/packages/openc3-tool-common/src/tools/base/AppNav.vue:183]: http/https agnostic urls.
//openc3.com

File: openc3-init/plugins/packages/openc3-tool-common/src/tools/base/AppNav.vue (hunk 2 above)
Comment (ryanmelt) [reply openc3-init/plugins/packages/openc3-tool-common/src/tools/base/AppNav.vue:188]: Parameters after a ? in the url

**Pull request merged**